roles/flannel/tasks/main.yml

```diff
@@ -1,9 +1,11 @@
 ---
 - name: Install nhorman Flannel yum repo file
   copy: src=flannel.repo dest=/etc/yum.repos.d/
+  when: not is_atomic
```

Comment:
Looks like i still have repos for rhel7, guess i should do the same here.  need to make this rhel7 only